src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -711,10 +711,7 @@ public function generateStepsPhp($actionObjects, $generationScope = TestGenerato
             }
 
             if (isset($customActionAttributes['function'])) {
-                $function = $this->addUniquenessFunctionCall(
-                    $customActionAttributes['function'],
-                    $actionObject->getType() !== "executeInSelenium"
-                );
+                $function = $this->addUniquenessFunctionCall($customActionAttributes['function']);
```

Comment:
Should we use $function = $this->addUniquenessFunctionCall($customActionAttributes['function'], false); ?

Reply:
it should be true, which is the default value if you look at addUniquenessFunctionCall's signature. Didn't explicitly set it to true to follow same format. eg: 
>  $html = $this->addUniquenessFunctionCall($customActionAttributes['html']);

Reply:
I think you are right.